pkgs/development/python-modules/dicom2nifti/default.nix

```diff
@@ -29,34 +34,49 @@ buildPythonPackage rec {
     hash = "sha256-lPaBKqYO8B138fCgeKH6vpwGQhN3JCOnDj5PgaYfRPA=";
   };
 
+  patches = [
+    # Hotfix for the OverflowError in tests and package usage.
+    # TODO(@osbm): Remove this patch when the upstream is fixed.
+    # https://github.com/icometrix/dicom2nifti/pull/152
+    (fetchpatch {
+      url = "https://github.com/icometrix/dicom2nifti/pull/152/commits/dac9ff492bec6d13157e5712d774ec1ea349abff.patch";
+      name = "fix-tests.patch";
```

Comment:
I don't think this should be called "fix-tests" but more likely something like "fix-integer-scaling" ?